src/Compilers/Core/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -416,14 +416,14 @@ internal static string RemoveTrailingSpacesAndDots(string path)
             out List<string> parsedArgs,
             out bool containsShared,
             out string keepAliveValue,
-            out string sessionKey,
+            out string pipeName,
```

Comment:
This is always a pipe name. In the past it was called "session key" because we thought there may be more than one IPC mechanism.